example-games/src/main/java/org/menacheri/zombie/game/ZombieSpringConfig.java

```diff
@@ -72,11 +72,14 @@
 	public @Bean World world()
 	{
 		World world = new World();
-		world.setAlive(2000000000);
-		world.setUndead(1);
-		return world;
+		world.setAlive(2000010000);
+		if (world.setUndead(1);){
```

Comment:
Any reason why there is a semicolon inside the if loop

Reply:
oops, didn't see that  you can take out the semicolon.